cmake/BuildBenchmark.cmake

```diff
@@ -0,0 +1,60 @@
+set(BENCHMARK_VERSION v1.9.0)
+set(BENCHMARK_HASH 21a2604efeded8b4cbabc72f3e1c7a2a)
+set(BENCHMARK_INSTALL_DIR ${BUNDLED_LIBS_INSTALL_DIR}/benchmark-prefix)
+set(BENCHMARK_INCLUDE_DIR ${BENCHMARK_INSTALL_DIR}/dest/include)
+
+set(BENCHMARK_LIB ${BENCHMARK_INSTALL_DIR}/build/src/libbenchmark.a)
+set(BENCHMARK_LIB_MAIN ${BENCHMARK_INSTALL_DIR}/build/src/libbenchmark_main.a)
+
+list(APPEND BENCHMARK_CMAKE_FLAGS
+     "-DBENCHMARK_ENABLE_TESTING=OFF"
+     "-DBENCHMARK_ENABLE_LTO=OFF"
+     "-DBENCHMARK_USE_LIBCXX=OFF"
+     "-DBENCHMARK_ENABLE_GTEST_TESTS=OFF"
+     "-DBENCHMARK_ENABLE_LIBPFM=OFF"
+     # By default, this library is built in Debug. Use the
+     # non-Debug build for benchmarks.
+     "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
+)
+
+include(ExternalProject)
+ExternalProject_Add(bundled-benchmark-project
+    PREFIX ${BENCHMARK_INSTALL_DIR}
+    SOURCE_DIR ${BENCHMARK_INSTALL_DIR}/src
+    BINARY_DIR ${BENCHMARK_INSTALL_DIR}/build
+    STAMP_DIR ${BENCHMARK_INSTALL_DIR}/stamp
+    URL_MD5 ${BENCHMARK_HASH}
+    DOWNLOAD_EXTRACT_TIMESTAMP TRUE
+    URL https://github.com/google/benchmark/archive/refs/tags/${BENCHMARK_VERSION}.tar.gz
+    CONFIGURE_COMMAND
+        ${CMAKE_COMMAND} -B <BINARY_DIR> -S <SOURCE_DIR>
+            -G ${CMAKE_GENERATOR} ${BENCHMARK_CMAKE_FLAGS}
+    BUILD_COMMAND cd <BINARY_DIR> && ${CMAKE_MAKE_PROGRAM} -j
```

Comment:
`-j` will fail with Ninja
probably we can ignore and fix this for all cmake modules after introducing `cmake/SetBuildParallelLevel.cmake` (https://github.com/tarantool/tarantool/pull/10216)

Reply:
Probably it is better to fix at all places at once and keep them consistent before. Ignoring for now.

Reply:
Agree.